help/archive-a-stream.md

```diff
@@ -13,28 +13,46 @@ messages and topics in the stream may or may not continue to work.
 In most cases, we recommend [renaming streams](/help/rename-a-stream) rather
 than archiving them.
 
-### Archive a stream
-
 {start_tabs}
 
 {relative|stream|all}
 
 1. Select a stream.
 
-1. Click the <i class="fa fa-trash-o"></i> on the right.
+1. Click the **trash** <i class="fa fa-trash-o"></i> icon near the top right
+   corner of the stream settings panel.
 
 1. Approve by clicking **Confirm**.
 
 {end_tabs}
 
 !!! warn ""
 
-    Archiving a stream is currently irreversible, but we plan to add support
-    for unarchiving streams in the future.
+    Archiving a stream is currently irreversible via the UI. If you are
+    self-hosting, you can unarchive a stream using the `reactivate_stream`
+    [management command](https://zulip.readthedocs.io/en/latest/production/management-commands.html#other-useful-manage-py-commands),
+    which will make it public, invite-only, and subscribe all organization
+    owners to it.
```

Comment:
Might be better to move this to a separate "Unarchiving archived streams" section so we can link to it.

I think we also probably want a sentence that Zulip Cloud users can contact us.